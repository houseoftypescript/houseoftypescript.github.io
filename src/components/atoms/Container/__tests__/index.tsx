import { render } from '@testing-library/react';
import { Container } from '../index';

describe('Container', () => {
  it('renders correctly', () => {
    const button = render(<Container></Container>);
    expect(button.container).toMatchSnapshot();
  });
});
