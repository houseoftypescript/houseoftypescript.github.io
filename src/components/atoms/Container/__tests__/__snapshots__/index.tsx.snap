// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container renders correctly 1`] = `
<div>
  <div
    class="container mx-auto px-4 sm:px-8"
  />
</div>
`;
